### PR TITLE
Expose ISteamGameServer::SetGameData

### PR DESCRIPTION
**Pull request by psychonic**
Let's have some fun.

### CODE REVIEW

File: Extension/gsnatives.cpp [hunk 1]
```diff
@@ -88,6 +88,23 @@ static cell_t sm_IsLoaded(IPluginContext *pContext, const cell_t *params)
...
+	}
+	
+	char *pData;
+	pContext->LocalToString(params[1], &pData);
+	
+	pServer->SetGameData(pData);
+	return 1;
+}
+
+
```
Comment (KyleSanderson) [inline Extension/gsnatives.cpp]: Is this intended?

Comment (psychonic) [reply Extension/gsnatives.cpp]: Yes, I explicitly added an extra newline. My hopes were to trigger you.

Comment (KyleSanderson) [reply Extension/gsnatives.cpp]: Can you please resolve this blocker?

Comment (psychonic) [reply Extension/gsnatives.cpp]: No can do. Sorry, brotato chip.

**Pull request merged**